src/lib/operators/join_sort_merge.cpp

```diff
@@ -132,6 +132,9 @@ class JoinSortMerge::JoinSortMergeImpl : public AbstractReadOnlyOperatorImpl {
   std::map<RowID, bool> _left_row_ids_emitted{};
   std::map<RowID, bool> _right_row_ids_emitted{};
```

Comment:
I replaced the `std::map` with `ska::bytell_hash_map` .

Reply:
Do you have some performance numbers for that? Generally, I suggest to only use those "special" hashmaps if they have a measurable performance benefit over `unordered_map`.

Reply:
(And yes, I probably should have included that information in the first comment. I'm somewhat unhappy with my own review here, sorry.)

Reply:
I replaced `ska::bytell_hash_map` with `std::unordered_map `. I think to compare the performance of these two, it could make sense to write a little benchmark so that both hash maps are using the same hash function. Do you think that would be helpful? Or is a `std::unordered_map ` sufficient, since the use of the maps is an edge case anyway?

Reply:
Works for me. If we come back and see that this is an issue, we can still look into other maps.